### ISSUE TITLE
[6] Dynamic icon importing: Cannot find module '@fortawesome/fontawesome-svg-core/import.macro'

### ISSUE THREAD

**Issue by shehi** [FortAwesome/Font-Awesome#18144]
Dynamic icon importing doesn't work... Any idea why? I properly have set both `.babelrc` macro setup and `babel-plugin-macros-config.js` file.

![image](https://user-images.githubusercontent.com/653569/128532297-2e7d0d02-e862-482a-b130-07d2ba2440bf.png)

**Comment by tagliala** [comment]
Hi!

Could you please provide a reproducible test case?

**Comment by shehi** [comment]
Already contacted you guys via help@fontawesome.com and someone called Trevor got 2-3 emails from me. I haven't received any responses after I told him my app is NextJs and then he redirected me to NuxtJs documentation. Should I post screenshots of my mails here? There can't be a test-case for this, I am not sure if you read what I just wrote above. Module isn't there, and you want somehow to test... what exactly?

**Comment by shehi** [comment]
Ok, guess you guys checked my ticket right after responding here. Someone, without a signature, responded: "Thank you for reporting this. You are correct and we'll get this on the list to be fixed!"

I think I should post my findings here as well for future reference, so here we go:

**EMAIL 1**

![image](https://user-images.githubusercontent.com/653569/128565193-e06f4bfe-67f2-4cda-b1ed-c12e41741c03.png)



**EMAIL 2**

![image](https://user-images.githubusercontent.com/653569/128565287-23a2b72f-5f5b-4ee6-8e38-14ab0c9c65bf.png)



**EMAIL 3**

![image](https://user-images.githubusercontent.com/653569/128565326-c6d14b27-61f2-4d27-8455-3f43489d531d.png)
![image](https://user-images.githubusercontent.com/653569/128565404-32bcc5f5-b4ac-46db-952a-711119f9a11e.png)

**Comment by tagliala** [comment]
> "Thank you for reporting this. You are correct and we'll get this on the list to be fixed!"

Great, thanks

> Module isn't there, and you want somehow to test... what exactly?

For future references: a repository with the minimum amount of code to reproduce the issue would have help in debugging. There are a lot of different ways of using Font Awesome. Without details it is hard to help

**Comment by shehi** [comment]
Guys, any news on this one?

**Comment by robmadole** [comment]
Answered via email. Closing here.

**Comment by robmadole** [comment]
If anyone else runs into this it's because we have an issue with how we've versioned the `@fortawesome/fontawesome-svg-core` package.

To fix it for the 6.0.0-beta1 release pin the version of `@fortawesome/fontawesome-svg-core` to `1.2.33-beta`.

In you `package.json` file it would look like this:

```javascript
"devDependencies": {
    "@fortawesome/fontawesome-svg-core": "1.2.33-beta1"
  }
```
